puppeteer/bundleRequests.test.js

```diff
@@ -1,4 +1,5 @@
 import puppeteer from 'puppeteer';
+import { localBaseUrl } from '../src/testHelpers/config';
```

Comment:
Is it worth trying to use the url defined in the local.env file? Not a major concern but would mean 1 less place to change this url.

Reply:
IMO its not worth it, would have to load a file a lot for each test